src/api-doc/api-doc.yaml

```diff
@@ -3698,14 +3698,14 @@ paths:
             application.json:
               schema:
                 $ref: '#/components/schemas/questions/readQuestionResponse400'
-  /mentoring/v1/questionsSet/create:
+  /mentoring/v1/question-set/create:
```

Comment:
@sumanvpacewisdom have you checked postman collection?

Reply:
yes @rakeshSgr , postman its updated , doc is not updated